src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -304,29 +304,27 @@ public ReactionPaginationAction retrieveReactionUsers(@Nonnull String unicode)
     }
 
     @Override
-    public MessageReaction.ReactionEmote getReactionByUnicode(@Nonnull String unicode)
+    public MessageReaction getReactionByUnicode(@Nonnull String unicode)
     {
         Checks.notEmpty(unicode, "Emoji");
         Checks.noWhitespace(unicode, "Emoji");
 
         return this.reactions.stream()
-            .map(MessageReaction::getReactionEmote)
-            .filter(r -> r.isEmoji() && r.getEmoji().equals(unicode))
+            .filter(r -> r.getReactionEmote().isEmoji() && r.getReactionEmote().getEmoji().equals(unicode))
             .findFirst().orElse(null);
     }
 
     @Override
-    public MessageReaction.ReactionEmote getReactionById(@Nonnull String id)
+    public MessageReaction getReactionById(@Nonnull String id)
```

Comment:
I saw you already mentioned this on discord, can you make this a default in the interface?